docs/docs/08.1-more-about-refs.md

```diff
@@ -9,6 +9,10 @@ After building your component, you may find yourself wanting to "reach out" and
 
 Let's look at how to get a ref, and then dive into a complete example.
 
+> Important:
+>
+> `ref` attributes do **not work** on [stateless/functional components](/react/docs/reusable-components.html#stateless-functions)
```

Comment:
Suggest changing this to "stateless functional components" since that is what seems to be used in other docs.